src/renderer/shared/i18n/locales/en.json

```diff
@@ -187,6 +187,8 @@
     },
     "successMessage": "Wallet added",
     "thresholdDescription": "Set the threshold of signatories required to execute a transaction",
+    "thresholdErrorTitle": "Invalid threshold",
+    "thresholdErrorDescription": "Threshold must be set and >= 2",
```

Comment:
Is it confirmed translation? Looks like `>=` can be not clear for users.

Reply:
It's not confirmed, I can change for "superior than 2"